src/SeqCli/Csv/CsvTokenizer.cs

```diff
@@ -26,9 +26,9 @@ protected override IEnumerable<Result<CsvToken>> Tokenize(TextSpan span)
                     if (!next.HasValue) yield break;
 
                     var text = Content(next.Location);
-                    while (text.HasValue)
+                    while (text.HasValue || !text.Remainder.IsAtEnd)
```

Comment:
This was necessary to deal with the change from `While()` to `WithoutAny()` a few lines earlier. The former created zero-length matches, while the latter doesn't.

We can revert to the earlier approach when something like `Span.MatchedBy(Character.Except('"').Many())` can be expressed in Superpower.